### PR TITLE
Removed protobuf dependency

### PR DESCRIPTION
**Pull request by utkarsharma2**
Removed protobuf dependency

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -64,7 +64,7 @@ google =
     apache-airflow-providers-google>=8.1.0
     gcloud-aio-storage
     gcloud-aio-bigquery
-    protobuf<=3.20.0  # Bigquery provider isn't compatible with it. Details in https://github.com/apache/airflow/commit/25a9ae3b2eec85dfd500b0a921045fc95ab8ffd6
+    protobuf  # Bigquery provider isn't compatible with it. Details in https://github.com/apache/airflow/commit/25a9ae3b2eec85dfd500b0a921045fc95ab8ffd6
```
Comment (pankajkoti) [inline setup.cfg:67]:
```suggestion
    protobuf 
```

**Review by pankajkoti — COMMENTED** [review]
Can we update the Title that we're not actually removing the dependency but removing the pin to an upper bound.
Also, would be nice to update the PR description on why we're removing the pin now and whether something has been fixed that was an issue earlier.

File: astronomer/providers/databricks/operators/new.py
Comment (Lee-W) [inline astronomer/providers/databricks/operators/new.py:1]: May I know why we need this file in this PR? Thanks!

**Comment by utkarsharma2** [comment]
@pankajkoti @Lee-W This PR was raised for debugging. We don't need this. Closing it.

**Pull request closed without merging**